packages/core/src/create/release.ts

```diff
@@ -157,7 +157,9 @@ export const createRelease = async (
     { releaseDetails, appDetails, publisherDetails },
     publisher.publicKey
   );
-  validateRelease(releaseJson);
+
+  const suppressedJson = JSON.stringify(releaseJson, metaplexFileReplacer, 2);
+  validateRelease(JSON.parse(suppressedJson));
```

Comment:
What issue is this addressing? I'm concerned with this usage of `metaplexFileReplacer` - as written, it's meant for debug output, and while this may work, it'll be brittle.

Reply:
Validation fails if we don't replace out the `Metaplex` file objects that get created at an earlier step in the process. FWIW, this was used elsewhere before this section of code.

Reply:
If the `MetaplexFile` objects are being removed, what are we really validating here? Is it representative of the final data, or just something convenient which will work with validation?

My concern here is that the `MetaplexFile` objects are being replaced with `"(suppressed)"` - if this passes schema validation, it's only coincidence. Fundamentally, `metaplexFileReplacer` was written for debug purposes, and we're overloading it to have two meanings. I'd rather duplicate it, once per file that consumes it, and make each purpose-built for that use case. For e.g., in this use case I'd replace `MetaplexFile` objects with something like `https://temp-asset-path`, so that if we do URI validation on the field in JSON in the future, validation logic would still pass because the replacer is representative.

Reply:
The honest truth is that feel like our validation logic is pretty hard to comprehend and not entirely clear in methodology. The code itself is understandable enough, but I can't quite derive a complete picture of _why_ it was implemented as-is. Feels like we're using validation for more than it was intended for and at odd times in the flow.

The _only_ field that gives validation problems is the `uri` field, and that _only_ occurs because at one step in our code it is replaced for Metaplex purposes. I think we should do as little work as possible on this right now and then re-evaluate the entire operational and figure out refactors. So, for now I'm just going to have it use the temp asset URI and that should be fine for now.